src/main/AndroidManifest.xml

```diff
@@ -28,7 +28,7 @@
         android:allowBackup="true"
         android:debuggable="false"
         android:icon="@drawable/icon"
-        android:label="WhereYouGo"
+        android:label="@string/app_name"
```

Comment:
Good idea to externalize it.
Please create and use a file strings_not_translatable.xml like cgeo.
And use a separate PR, it is not a part of this PR.

Reply:
See PR #84. I want to rebase this PR on the other, is this okay and how can I do it? Is it sufficient to run "git rebase strings_not_translatable" on my local branch, then do a "git push --force"?

Reply:
After local rebase, the commit history and changes in the commit for this pr are looking good.